pvlib/pvsystem.py

```diff
@@ -609,6 +609,33 @@ def faiman_celltemp(self, poa_global, temp_air, wind_speed=1.0):
         return temperature.faiman(poa_global, temp_air, wind_speed,
                                   **kwargs)
 
+    def fuentes_celltemp(self, poa_global, temp_air, wind_speed):
+        """
+        Use :py:func:`temperature.fuentes` to calculate cell temperature.
+
+        Parameters
+        ----------
+        poa_global : pandas Series
+            Total incident irradiance [W/m^2]
+
+        temp_air : pandas Series
+            Ambient dry bulb temperature [C]
+
+        wind_speed : pandas Series
+            Wind speed [m/s]
+
+        Returns
+        -------
+        temperature_cell : pandas Series
+            The modeled cell temperature [C]
+        """
+        kwargs = _build_kwargs([
+            'noct_installed', 'module_height', 'wind_height', 'emissivity',
+            'absorption', 'surface_tilt', 'module_width', 'module_length'],
```

Comment:
I think `'surface_tilt'` is a `PVSystem` attribute, not to be found in `temperature_model_parameters`

We might also consider getting `'module_width'` and `'module_length'` from `module_parameters`, although: the database doesn't supply these values and they aren't used anywhere else. `'module_height'` is really an attribute of the PV System.

For now, I'm OK expecting the `'module_'` values in `temperature_model_parameters` but they will move once another, non-temperature function needs this information - e.g., a bifacial irradiance function.

Reply:
Valid points.  I unthinkingly pulled all the extra parameters from `temperature.fuentes` and dumped them there.  

I can imagine some people objecting to using Fuentes with the real `surface_tilt` value because they want the same behavior as the SSC implementation of PVWatts, which hardcodes the tilt at 30.  Would it make sense to use the `PVSystem` attributes by default but allow the user to override them by providing alternate values in the `temperature_model_parameters` dictionary?

Reply:
I don't have a better idea at the moment, than using  `temperature_model_parameters['surface_tilt']` as the signal to override. But I think it risks confusing users, to have two 'surface_tilt' quantities with different meanings. Maybe a Note in the `PVSystem` docstring and move ahead.

Reply:
Also something to consider when creating an Array class.